app/src/androidTest/java/org/simple/clinic/patient/businessid/BusinessIdMetaDataAdapterAndroidTest.kt

```diff
@@ -41,6 +41,10 @@ class BusinessIdMetaDataAdapterAndroidTest {
                 assigningUserUuid = UUID.randomUUID(),
```

Comment:
We are adding the meta data type to the `BusinessId` type and not to the table, so the commit message should reflect this. Can we reword the commit, please?